confluence/client.py

```diff
@@ -463,6 +463,27 @@ def get_attachments(self, content_id, filename=None, media_type=None, expand=Non
                                        params=params,
                                        expand=expand)
 
+    def download_attachment(self, attachment):
+        # type: (Content) -> [byte]
+        """
+        Downloads an attachment. To be used in conjunction with get_attachments
+
+        :param attachment: A content record for the attachment to be downloaded
+        
+
+        :return: A byte array for the attachment.
+        """
+
```

Comment:
I'd like to see a defensive check here on content.type to ensure that we're actually using a real attachment object.

Reply:
Although maybe that should include/be `if 'download' not in attachment.links then bail`. Reasonably ambivalent about that